apps/explorer/lib/explorer/chain.ex

```diff
@@ -3439,7 +3439,7 @@ defmodule Explorer.Chain do
   def fetch_recent_collated_transactions_for_rap(paging_options, necessity_by_association) do
     fetch_transactions_for_rap()
     |> where([transaction], not is_nil(transaction.block_number) and not is_nil(transaction.index))
-    |> handle_random_access_paging_options(paging_options)
+    |> handle_random_access_paging_options(paging_options, false)
```

Comment:
What determines whether the second argument should be `true` or `false`?

Reply:
whether the records being fetched are fee_payments or forward_transfers (ie extra_transfers). this is necessary because transactions set a hard limit of of number of records it will display on the explorer. That was why we were seeing the duplicate entries in fp, because it had exceeded that limit.